docs/operations/clone_api.md

```diff
@@ -139,14 +140,13 @@ kubectl get vm vm-clone-target -o yaml
 The clone API is in an early alpha version and may change dramatically. There are many improvements and features
 that are expected to be added, the most significant goals are:
 
-* Add more supported source types. The most important is `VirtualMachineSnapshot` (see section below for more info). We
-  also intend to support `VirtualMachineInstace` types in the future.
+* Add more supported source types, like `VirtualMachineInstace` in the future.
 * Add a cross-namespace clone support. This needs to be supported for snapshots / restores first.
 
 ### Using clones as a "golden VM image"
 
-One of the great things that could be accomplished with the clone API when `VirtualMachineSnapshot`s are supported
-as source types is to create "golden VM images" (a.k.a. Templates / Bookmark VMs / etc). In other words, the following
+One of the great things that could be accomplished with the clone API when the source if of kind `VirtualMachineSnapshot`
```

Comment:
`the source if of kind`? Seems like a typo :)

Reply:
Yup! Indeed a typo :)
Fixed to `the source is of kind`